### DISCUSSION TITLE
Sign of the drift rate in the ADDM

### DISCUSSION THREAD

**Discussion by Jiashun97** [mwshinn/PyDDM#100] [Ask for help] [answered]
Hi Max,

Thanks for your recent update of the pyddm package. It looks great! I am now implementing the ADDM by customizing the drift rate.  I was wondering if the drift rate should be negative when `current_fixation == 0` for example in the follow script. If I am not adding the negative sign, I get some weird results of parameter estimates (such as d=0 or theta = 0 or 1). It would be great if you can share some insight here. 
<img width="603" alt="Screenshot 2024-06-19 at 17 27 32" src="https://github.com/mwshinn/PyDDM/assets/65221219/d0652d4e-2dcd-4c8a-8687-ed56a3fabdf1">

Best regards,
Jiashun

**Comment by mwshinn** [comment, marked as answer]
You're right, thanks for the bug report!  Fixed in the documentation with commit cf1eb2dea33a7efbccabc1a1f9342233f4c0fa20.
